bloomstack_core/bloomstack_core/page/admin_insights/admin_insights.py

```diff
@@ -0,0 +1,10 @@
+import frappe
+
+
+@frappe.whitelist()
+def get_cubejs_host():
+    print("function is calling from home")
```

Comment:
- Remove the print statements.
- Also how are you planning to put the credentials in the site config in the first place ?